client/src/Actions/UserActions.js

```diff
@@ -110,22 +153,24 @@ export const DeleteUser = ( token ) => dispatch => {
       dispatch({
         type: types.DELETING_USER_SUCCESS
       })
-      dispatch({
-        type: types.CLEAR_USER_FROM_STORE
-      })
+      // Since Deletion of an account immediately results in logging out, should the ENTIRE store be cleared?
+      // TODO: (?) Add general store reset action type to all reducers
+      Logout()
     })
     
     .catch(error => {
       dispatch({ 
         type: types.DELETING_USER_FAILURE,
-        payload: {error} 
+        payload: {
+          error
+        } 
       })
     })
 }
 
 // On logout, which can only be done through the UserProfile, clear the entire store by calling every individual CLEAR action
 // >>> Find a way to clear the entire store in a single go.
-export const Logout = credentials => dispatch => {
+export const Logout = () => dispatch => {
   localStorage.clear('token')
```

Comment:
We can use localStorage for now, but I think we should implement HTTP cookies in future. 👍

Reply:
Fully agree. As of right now, I want to focus on getting barebones routing and data access going. Aiming for the end of next week to implement cookies.